include/deal.II/lac/precondition.h

```diff
@@ -1173,22 +1188,6 @@ class PreconditionChebyshev : public Subscriptor
    */
   mutable Threads::Mutex mutex;
 
-  /**
-   * Runs the inner loop of the Chebyshev preconditioner that is the same for
-   * vmult() and step() methods.
-   */
-  void
-  do_chebyshev_loop(VectorType &dst, const VectorType &src) const;
-
-  /**
-   * Runs the inner loop of the Chebyshev preconditioner that is the same for
-   * vmult() and step() methods. Uses a separate function to not force users
-   * to provide both vmult() and Tvmult() in case only one variant is
-   * requested in subsequent calls.
-   */
-  void
-  do_transpose_chebyshev_loop(VectorType &dst, const VectorType &src) const;
-
```

Comment:
For someone reviewing this part: We had previously put the code for `vmult()` and `step()` as well as `Tvmult()` and `Tstep()` into these common functions. Given some changes, I would now also have to pass the iteration index. Given that each function only calls two other functions and does an update to a scalar, I have put the code inline because it does not save us any lines in the end.